pyproject.toml

```diff
@@ -81,6 +81,7 @@ docs = ["recommonmark", "sphinx", "sphinx-click", "sphinx_rtd_theme", "sphinxcon
 types = ["types-click", "types-chardet", "types-setuptools", "pyspark-stubs", "types-filelock", "types-PyYAML"]
 
 [tool.poetry.dev-dependencies]
+attrs = ">=21.1"
```

Comment:
but then I think by default this will get install nonetheless, so it will override with line 50

Reply:
Yes, but it won't be an issue if user packages require old attrs for some reason.

Reply:
(really detailed)